scoap3/utils/google_maps.py

```diff
@@ -1,20 +1,34 @@
 import requests
 from flask import current_app
+from invenio_db import db
+from sqlalchemy import func
+from sqlalchemy.orm.exc import NoResultFound
 
-from scoap3.dojson.utils.nations import NATIONS_DEFAULT_MAP
+from scoap3.config import COUNTRIES_DEFAULT_MAPPING
+from scoap3.modules.analysis.models import CountryCache
 
 
-def __get_country_from_results(results):
-    if 'results' in results:
-        for address_component in results['results'][0]['address_components']:
-            if 'country' in address_component['types']:
-                return address_component['long_name']
+def get_country(text):
+    """
+    Tries to get find the country for the given text. It can use multiple Google API calls to decide.
```

Comment:
Just a typo. I think you wanted to have this version: Tries to find the country for the given text.